packages/snjs/lib/models/app/search.ts

```diff
@@ -0,0 +1,40 @@
+/** Grouping of required state variables to perform the search */
+
+import { CollectionSort } from "@Lib/protocol/collection/item_collection";
+import { SNTag } from "./tag";
+
+export interface SearchPayload {
```

Comment:
A payload is a low-level representation of data that is generally in transit, like from server to app, or from app to disk. Did you have other names in mind? `SearchOptions` seems the most obvious one to me?

Reply:
`SearchQuery` is a good option.

Reply:
I'm also not seeing why there is a need to separate `SearchPayload` and `Search` into two definitions. Should just be one object, `SearchQuery`.

Reply:
You're right that was unnecessary overhead.